src/api/VotingIrregularities.Domain/Models/FormVersion.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace VotingIrregularities.Domain.Models
+{
+    public partial class FormVersion
+    {
+        [Key, Required, MaxLength(2)]
```

Comment:
@ExtremeSwat is there any particular reason why you changed the property here from "Code" to "Id"? 
Is the controller exposing this model change?
My concern is that we are breaking integration with older versions of the mobile app this way. Can we add the Id property, as well as keeping the Code property?

Reply:
The only reason for the change was due to personal coding preference, now the respective field should't interfere with any existing 'client' (in terms of model mismatch) since they've only been parsing dictionaries up until now from the [versiune] endpoint which wasn't exposing this property name.
Anyways, I've added a decorator for that Id property to have it as 'Code' for the front-end.